Logo/bin/generate-assets.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+wd=`pwd`
+destdir="autogenerated"
+pngwidths="300 600 1200"
+totalassets=6
+
+find . -type f | grep svg >/dev/null || {
+	echo "No suitable files found. Sure we are in the right dir?"
+	return
```

Comment:
This should be `exit 1` - any time the script fails, it should exit with a non-zero exit code.

Reply:
Done!